src/gradualizer_lib.erl

```diff
@@ -193,6 +193,8 @@ pick_value(?type(nonempty_list, Ty), Env) ->
     {cons, erl_anno:new(0), H, {nil, erl_anno:new(0)}};
 pick_value(?type(nil), _Env) ->
     {nil, erl_anno:new(0)};
+pick_value(?type(binary, _), _Env) ->
+    {bin, erl_anno:new(0), []};
```

Comment:
`<<>>` is not a value of every bitstring type. The type might have a fixed or minimum length >= 0.

Reply:
Updated. There's no distinction between minimum and fixed length binaries when printing, but there's a special case for empty binaries.